File: Chibuzor tasks/Arrays snacks/LargestElement.java
public class LargestElement{
	public static void main(String[] args){
	
	int [] numbers = {12,23,43,55,76};

	int large = array(numbers);
	
	System.out.print(large);
}
public static int array(int[] numbers){

	int largest = numbers[0];

	for(int number = 0; number < numbers.length; number++){
		if(numbers[number] > largest){
			largest = numbers[number];

		}
	}
	return largest;
	}
}

File: Chibuzor tasks/Arrays snacks/largest_element.py
def largest_of(numbers[]):
	largest = numbers[0]
	for index in numbers:
		if numbers[index] > largest:
			largest = numbers[index]
	return largest


arr_values = {1,2,3,4,5,6,7}
result = largest_of(arr_values)
print(result)